kbase-extension/static/kbase/custom/custom.js

```diff
@@ -301,9 +301,9 @@ define([
         })();
 
         p.renderMinMax = function () {
-            let $cellNode = $(this.element),
-                metaToggleMode = utils.getCellMeta(this, 'kbase.cellState.toggleMinMax'),
-                toggleMode = $cellNode.data('toggleMinMax');
+            const $cellNode = $(this.element),
```

Comment:
Fixed all the eslint errors in this file whilst I was looking for some other info.